base/file.jl

```diff
@@ -454,6 +563,11 @@ function symlink(p::AbstractString, np::AbstractString)
     uv_error("symlink",err)
 end
 
+"""
+    readlink(path::AbstractString) -> AbstractString
+
+Returns the value of a symbolic link `path`.
```

Comment:
(`readlink` docstring) is "the value of a symbolic link" unambiguous? maybe worth rewording to say something like "the target location a symbolic link points to" ?